File: Modul 4/C/PRAK402.c
#include <stdio.h>
int main() {
    int bilangan;
    printf("Input\n");
    scanf("%d", &bilangan);
    printf("Output\n");

    for (int i = 1; i <= bilangan; i++) { 
        //i <= bilangan menyatakan loop akan terus berjalan selama i <= bilangan
        if (i % 2 != 0) {
            printf("%d ", i);
        }
    }
    printf("\n");
    for (int i = bilangan; i>= 1; i--) {
        if (i % 2 == 0) {
            printf("%d ", i);
        }
    }
    return 0;
}

File: Modul 4/PYTHON/PRAK402.py
print("Input")
bilangan = int(input())
print("Output")

for i in range(1, bilangan+1): 
#bilangan+1 digunakan sebagai rentang. 
# jadi akan melakukkan looping dari 1 hingga bilangan itu sendiri.
        if (i % 2 != 0):
            print(i, end=" ")
print()
for i in range(bilangan, 1, -1):
        if (i % 2 == 0):
            print(i, end=" ")